wp-media/wp-rocket
# Added pdf extension as well as doc|docx|xls to image_ext var

## Description

Added pdf|doc|docx|xls to image_ext variable to exlude from preload
Fixes #3720

## Type of change



## Is the solution different from the one proposed during the grooming?

This solution is in no way different from the proposed one.

## How Has This Been Tested?

- Enable preload link in WPR
- Add pdf|doc|docx|xls file to media and add link to a page.
- Open browser network tab and hover over file link to see that link is not preloaded.


# Checklist:
